src/Client.php

```diff
@@ -77,6 +77,20 @@ class Client
      */
     private $rabbitMqObjects = [];
 
+    /**
+    * Maximum connection's retry
+    **/
+    const MAX_RETRY = 10;
```

Comment:
This should not be set to a "const".
This is something one should be able to configure.

Also, by default, we should stay on the current behaviour (i.e.: NO retry)

This is **very important** because if your scripts pushing events start pausing for 300 seconds when there is a RabbitMQ failure, the user simply needs to continuously press the F5 key in its browser to fill your server's RAM with PHP processes awaiting the RabbitMQ connection.

This "retry" mechanism should be limited to only "listening" processes.